contents/uix/accessibility.md

```diff
@@ -0,0 +1,181 @@
+<frontmatter>
+  title: Accessibility
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Accessibility
+
+**Author(s): [Monika Manuela Hengki](https://github.com/monmanuela)**
+
+Reviewers: [Rahul Rajesh](https://github.com/rrtheonlyone), [Jacob Li Pengcheng](https://github.com/jacoblipech), [Tan Heng Yeow](https://github.com/tanhengyeow)
+
+## What is accessibility?
+
+> Accessibility is the practice of making your websites usable by as many people as possible — we traditionally think of this as being about people with disabilities, but really it also benefits other groups such as those using mobile devices, or those with slow network connections. <sub>--[MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)</sub>
+
+Note that although the scope of this article is the accessibility of websites, many of the points also apply to accessibility and UI/UX of other softwares too.
```

Comment:
This could be placed in a box and just above heading `## What is accessibility?` as it applies to the entire article.

```
<box type="info">

Note that although the scope of this article is the accessibility of websites, many of the points also apply to accessibility and UI/UX of other softwares too.
</box>
```